assembly/src/ast/parsers/debug.rs

```diff
@@ -0,0 +1,36 @@
+use super::{
+    parse_checked_param,
+    Instruction::*,
+    Node::{self, Instruction},
+    ParsingError, Token,
+};
+use vm_core::DebugOptions;
+
+// INSTRUCTION PARSERS
+// ================================================================================================
+
+/// Returns `Debug` instruction node.
+///
+/// # Errors
+/// Returns an error if the instruction token contains a wrong number of parameters, or if
+/// the provided parameters are not valid.
+pub fn parse_debug(op: &Token) -> Result<Node, ParsingError> {
+    debug_assert_eq!(op.parts()[0], "debug");
+    if op.num_parts() < 2 {
+        return Err(ParsingError::missing_param(op, "debug.[stack|mem|local].<debug_params?>"));
```

Comment:
It looks like we only support `stack` at the minute so may we remove `mem` and `local` from this error until they're introduced.